torch/_inductor/codegen/cuda/cutlass_utils.py

```diff
@@ -379,7 +387,13 @@ def get_accumulator_dtype(
         ]:
             torch_dtype = dtype0
 
-    if torch_dtype in (torch.float16, torch.bfloat16, torch.float, torch.float8_e4m3fn):
+    if torch_dtype in (
+        torch.float16,
+        torch.bfloat16,
+        torch.float,
+        torch.float8_e4m3fn,
+        torch.float8_e5m2,
+    ):
```

Comment:
this change (implicitly) would allow e5m2 FP8 gemm which would not be seen in most cases.
An alternative I considered is to assert that if e5m2 is in input_torch_dtypes, the other is e4m3.